packages/postcss-normalize-positions/src/index.js

```diff
@@ -140,12 +146,10 @@ function transform(value) {
         nodes[2].value = nodes[1].value = '';
       }
 
-      const map = Object.assign({}, horizontal, {
-        center,
-      });
+      const map = new Map([...horizontal, ['center', center]]);
```

Comment:
The test I have added should add coverage for this line. Previously, all tests passed even when I had made a mistake here.